pandas/tseries/tests/test_resample.py

```diff
@@ -1118,6 +1118,26 @@ def test_resample_loffset_count(self):
 
         assert_series_equal(result, expected)
 
+    def test_resample_loffset_agg(self):
+        # GH 13218
+        start_time = '1/1/2000 00:00:00'
+        rng = date_range(start_time, periods=50, freq='S')
+        vals = np.random.randn(50)
+        df = DataFrame({'A': vals, 'B': vals}, index=rng)
+
+        result = df.resample('10S', loffset='2h').agg({'A': 'count',
+                                                       'B': 'mean'})
```

Comment:
@jreback @sinhrks I was way too naive with the fix, it ends up applying loffset twice when not passing a dict to agg(). I will try to get a better fix along with more tests tonight.